pom.xml

```
@@ -0,0 +1,203 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.noumenadigital.platform</groupId>
+    <artifactId>npl-cli</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>${project.artifactId}</name>
+    <description>NPL CLI is a command-line tool designed to interact with the NPL Runtime and NOUMENA Cloud
+    </description>
+    <url>https://documentation.noumenadigital.com/</url>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <organization>Noumena Digital</organization>
+            <organizationUrl>https://noumenadigital.com/</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <url>https://github.com/NoumenaDigital/npl-cli</url>
+    </scm>
+
+    <properties>
+        <platform.version>2025.1.2</platform.version>
+        <kotlin.version>2.1.10</kotlin.version>
+        <kotest.version>5.9.1</kotest.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <commons-text.version>1.10.0</commons-text.version>
+        <slf4j-api.version>2.0.16</slf4j-api.version>
+        <maven-jar-plugin.version>3.7.1</maven-jar-plugin.version>
```

```suggestion
        <maven-jar-plugin.version>3.4.2</maven-jar-plugin.version>
```

You replicated an old mistake I made in the LS repo 😅

---

(3.4.2 is the actual latest, 3.7.1 does not exist)

---

interesting, how it was working then oO

---

Good question 😨